packages/cos/collection.yaml

```diff
@@ -2,20 +2,20 @@ packages:
   - &cos
     name: "cos"
     category: "system"
-    version: 0.8.2-6
+    version: 0.8.2-8
     description: "cOS base image, used to build cOS live ISOs"
     brand_name: "cOS"
     labels:
       autobump.revdeps: "true"
   - !!merge <<: *cos
     name: "cos-container"
     description: "cOS container image, used to build cOS derivatives from scratch"
-    version: 0.8.2-7
+    version: 0.8.2-8
```

Comment:
@mudler do you have an idea why these version literals keep appearing here? Is it caused by the autobump? I removed them already few times, it is quite annoying having to manually trace all this literals when we could enforce them to be equal and centralized.

Reply:
Correct :( it is the autobumper which doesn't understand yaml syntax properly and re-applies them anytime.. sadly that would require some work on it which is tracked in #842 and #654